src/components/ExcelDownloadButton.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+    <v-btn id="download-excel-btn" aria-label="Download Excel" icon density="compact" @click="download()">
+        <v-icon>mdi-download</v-icon>
+        <v-tooltip location="bottom" activator="parent">Download Excel</v-tooltip>
+    </v-btn>
+    <v-snackbar v-model="snackbarOpen" variant="outlined" color="error">
+        Error downloading Excel file: {{ downloadError.message || downloadError.toString() }}
+        <template v-slot:actions>
+            <v-btn icon @click="snackbarOpen = false">
+                <v-icon>mdi-close</v-icon>
+            </v-btn>
+        </template>
+    </v-snackbar>
+</template>
+
+<script setup lang="ts">
+import { ref, watch } from "vue";
+import { useExcelDownload } from "../composables/useExcelDownload";
+
+const { download, downloadError } = useExcelDownload();
+
+const snackbarOpen = ref(false);
+
+watch(downloadError, () => {
+    snackbarOpen.value = !!downloadError.value;
+});
+</script>
+
+<style lang="scss">
+.v-snackbar__wrapper {
+    background-color: rgba(255, 255, 255, 1) !important;
+}
```

Comment:
Snackbar showed up as annoyingly transparent without this, maybe there's a more elegant way to address it, but setting the opacity on the component didn't seem to work for me...

Reply:
i would have just done it as `variant="elevated" color="#a50026"` (example colour that i tried but any colour really), i believe the opacity didnt work because the actual color for `"error"` is itself faded